### PR TITLE
Added February 2022 Problems (Bronze - Gold)

### PR DESCRIPTION
**Pull request by jessechoe10**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: content/2_Bronze/Intro_Complete.problems.json [hunk 1]
```diff
@@ -147,6 +147,19 @@
...
+        {
+      "uniqueId": "usaco-1203",
+      "name": "Sleeping in Class",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1203",
+      "source": "Bronze",
+      "difficulty": "Hard",
+      "isStarred": true,
+      "tags": ["Complete Search"],
+      "solutionMetadata": {
+        "kind": "usaco",
```
Comment (bqi343) [inline content/2_Bronze/Intro_Complete.problems.json:160]: ^ needs to be "USACO"

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/3_Silver/DFS.problems.json [hunk 2]
```diff
@@ -124,6 +124,19 @@
...
       }
     },
+    {
+      "uniqueId": "usaco-1206",
+      "name": "Redistributing Gifts",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1206",
+      "source": "Silver",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Cycles", "Floyd Warshall"],
```
Comment (bqi343) [inline content/3_Silver/DFS.problems.json:134]:
```suggestion
      "tags": ["SCC"],
```

File: content/extraProblems.json [hunk 3]
```diff
@@ -705,6 +705,71 @@
...
       }
     },
+    {
+      "uniqueId": "usaco-1204",
+      "name": "Photoshoot 2",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1204",
+      "source": "Bronze",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Brute Force"],
```
Comment (bqi343) [inline content/extraProblems.json:715]: ^ why?

File: content/extraProblems.json [hunk 4]
```diff
@@ -705,6 +705,71 @@
...
+      }
+    },
+    {
+      "uniqueId": "usaco-1205",
+      "name": "Blocks",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1205",
+      "source": "Bronze",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["Brute Force"],
```
Comment (bqi343) [inline content/extraProblems.json:728]: "brute force" = complete search?

File: content/extraProblems.json [hunk 5]
```diff
@@ -705,6 +705,71 @@
...
+      }
+    },
+    {
+      "uniqueId": "usaco-1207",
+      "name": "Robot Instructions",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1207",
+      "source": "Silver",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Recursion", "Map", "Sorting", "Meet in the Middle"],
```
Comment (bqi343) [inline content/extraProblems.json:741]:
```suggestion
      "tags": ["Map", "Sorting", "Meet in the Middle"],
```

File: content/extraProblems.json (hunk 3 above)
Comment (jessechoe10) [reply content/extraProblems.json:715]: idk what should i put it as? would ad hoc be better?

File: content/extraProblems.json [hunk 6]
```diff
@@ -705,6 +705,71 @@
...
+        "kind": "USACO",
+        "usacoId": "1207"
+      }
+    },
+    {
+      "uniqueId": "usaco-1208",
+      "name": "Email Filing",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1208",
+      "source": "Silver",
+      "difficulty": "Normal",
```
Comment (bqi343) [inline content/extraProblems.json:752]: note - this should be "Hard" if the test data was stronger. it is not that difficult to pass this problem with an incorrect solution

Comment (jessechoe10) [reply content/extraProblems.json:752]: lol is that why i got 8/12 test cases?

File: content/extraProblems.json [hunk 7]
```diff
@@ -705,6 +705,71 @@
...
+        "kind": "USACO",
+        "usacoId": "1208"
+      }
+    },
+    {
+      "uniqueId": "usaco-1210",
+      "name": "Cow Camp",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1210",
+      "source": "Gold",
+      "difficulty": "Very Hard",
```
Comment (bqi343) [inline content/extraProblems.json:765]:
```suggestion
      "difficulty": "Hard",
```

File: content/extraProblems.json (hunk 3 above)
Comment (bqi343) [reply content/extraProblems.json]: just don't tag it at all? that's what we've been doing so far for ad hoc problems

**Comment by jessechoe10** [comment]
@bqi343 you should add the plat problems. i really dk what categories to put for those bc I'm only familiar with bronze-gold topics.

**Pull request merged**